dev-packages/e2e-tests/test-applications/sveltekit/test/performance.test.ts

```diff
@@ -36,6 +34,8 @@ test('captures a distributed pageload trace', async ({ page }) => {
     return txnEvent?.transaction === 'GET /users/[id]';
   });
 
+  await page.goto('/users/123xyz');
```

Comment:
should we add this to the `Promise.all` array underneath?

Reply:
Should be fine honestly.